### ISSUE TITLE
Crashes on start by failing to parse cherrypy version number

### ISSUE THREAD

**Issue by Isgar** [devsnd/cherrymusic#633]
Using Arch Linux and devel branch

Version numbering seems to differ on my system. The commit that broke it: https://github.com/devsnd/cherrymusic/commit/43856934a041bb65904d2459eb8bd120e4bdc706

```
Traceback (most recent call last):
  File "./cherrymusic", line 33, in <module>
    import cherrymusicserver
  File "/home/cherrymusic/cherrymusic/cherrymusicserver/__init__.py", line 167, in <module>
    cherrypy_version = tuple(int(v) for v in cherrypy.__version__.split('.'))
  File "/home/cherrymusic/cherrymusic/cherrymusicserver/__init__.py", line 167, in <genexpr>
    cherrypy_version = tuple(int(v) for v in cherrypy.__version__.split('.'))
ValueError: invalid literal for int() with base 10: 'post20160705'
```

For the time being, I have changed the line to `cherrypy_version = (5, 5)` which also solves #632 

Looks like the cherrypy version number is not reliable

**Comment by devsnd** [comment]
Ah shit, I tried [many different versions](https://travis-ci.org/devsnd/cherrymusic) on the continuous integration platform...

> Looks like the cherrypy version number is not reliable

Yeah, looks like it. I guess I should not rely on the version at all and instead try to figure out the version of the lib using other means :cry: 

Sorry for breaking your install twice in a row, I will release a fix version this evening! And thanks for reporting!

**Comment by Isgar** [comment]
Don't worry about breaking it, I'm glad I can help this awesome project in some way ;)

**Comment by devsnd** [comment]
Damn, I just didn't find the time to do the fix...
I have so little time, I didn't even update *my own* homeserver to the latest version yet :crying_cat_face: 

![screenshot - 18 07 2016 - 10 23 47](https://cloud.githubusercontent.com/assets/1980794/16909247/b9b255ba-4cd1-11e6-91c8-f023c0969252.png)

The fix is still somewhere near the top of my to-do list!

**Comment by devsnd** [comment]
@Isgar Damn, I am the worst software maintainer of all time. Sorry it took me three weeks to do this little fix! I released a new version containing the fix! (0.37.1)
